Docs/Book/Cartridge.md

```diff
@@ -65,29 +65,29 @@ Start by downloading and installing the postgresql dump from the ChEMBL website
 
 Connect to the database, install the cartridge, and create the schema that we'll use:
 
-    chembl_23=# create extension if not exists rdkit;
-    chembl_23=# create schema rdk;
+    chembl_25=# create extension if not exists rdkit;
+    chembl_25=# create schema rdk;
 
 Create the molecules and build the substructure search index:
 
-    chembl_23=# select * into rdk.mols from (select molregno,mol_from_ctab(molfile::cstring) m  from compound_structures) tmp where m is not null;
+    chembl_25=# select * into rdk.mols from (select molregno,mol_from_ctab(molfile::cstring) m  from compound_structures) tmp where m is not null;
     SELECT 1727081
```

Comment:
I think this line should probably also have changed when you updated to ChEMBL25

Reply:
updated